src/Commands/ImportProducts.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Str;
+use Rapidez\Core\Facades\Rapidez;
+use Rapidez\Statamic\Jobs\ImportCategoriesJob;
+use ReflectionClass;
+use Statamic\Facades\Entry;
+use Statamic\Facades\Site;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+class ImportProducts extends Command
+{
+    protected $signature = 'rapidez:statamic:import:products {--site=* : Sites handles or urls to process, if none are passed it will be done for all sites (Default will be all sites)}';
+
+    protected $description = 'Create a new product entry for a product if it does not exist.';
+
+    public function handle()
+    {
+        $productModel = config('rapidez.models.product');
+
+        $sites = $this->option('site');
+        $sites = $sites
+            ? array_filter(array_map(fn($handle) => (Site::get($handle) ?: Site::findByUrl($handle)) ?: $this->output->warning(__('No site found with handle or url: :handle', ['handle' => $handle])), $sites))
+            : Site::all();
+
+        $bar = $this->output->createProgressBar(count($sites));
+        $bar->start();
+        foreach($sites as $site) {
+            $bar->display();
+
+            $siteAttributes = $site->attributes();
+            if (!isset($siteAttributes['magento_store_id'])) {
+                continue;
+            }
+
+            Rapidez::setStore($siteAttributes['magento_store_id']);
+
+            $products = $productModel::query()
+                ->selectAttributes(['entity_id', 'sku', 'name', 'url_key'])
+                ->lazy();
+            
+            foreach ($products as $product) {
+                static::createEntry(
```

Comment:
What happens with the title when making the product content entry? Does it use the product name by default?

Reply:
Yes it does use the product name as title, this happens in the `RapidezStatamicServiceProvider` in the `bootListeners` function.
```php
Event::listen('rapidez-statamic:product-entry-data', fn($product) => [
     'title' => $product->name,
     'slug' => trim($product->url_key),
]);
```